hsm-crypto/src/main/java/com/webank/blockchain/hsm/crypto/sdf/AlgorithmType.java

```diff
@@ -0,0 +1,55 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 4.0.2
+ *
```

Comment:
Maybe it is better to create a new generated subdirectory and place the generated code in this directory.